templates/azure/index.html

```diff
@@ -13,13 +13,13 @@ <h1>
           Ubuntu on Azure
         </h1>
         <p>
-          Whether you are moving to Microsoft Azure or are already running cloud-native, Ubuntu is the platform of choice on Azure for virtual machine and container workloads.
+          Canonical produces official Ubuntu images in the Microsoft Azure Marketplace and continuously tracks and delivers updates to these, ensuring security and stability are built-in from the moment your machines and containers launch.
         </p>
         <p>
-          Canonical continuously tracks and delivers updates to Ubuntu images, ensuring security and stability are built-in from the moment your machines and containers launch.
+          Canonical works closely with Microsoft to optimize Ubuntu images on Azure and ensure Ubuntu supports the latest cloud features as these are released. Today, Ubuntu powers more mission-critical workloads on Azure than any other operating system.
```

Comment:
This isn't technically an error, but I've only seen us use British spelling so it would be "optimise" instead of "optimize"